### ISSUE TITLE
Profile image is missing alt text

### ISSUE THREAD

**Issue by srimanh** [kalviumcommunity/devlinks-01#2]
The avatar <img> in src/components/Header.jsx has no alt attribute. Add descriptive alt text.

**Comment by balaji206** [comment]
I'd like to work on this.
